src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature1Steps.ts

```diff
@@ -7,6 +7,6 @@ When("a visionary leader enters", world => {
    world.registerHandler("ReturnsEmptyPlanEventHandler")
    world.sendEvent(new node.Commit)
 })
-Then("excitement ensues", (p,world) => {
+Then("excitement ensues", world => {
```

Comment:
`HandlerScenarioWorld` returns `world` for both parameters. The first parameter is really "fixture". In testing projects, you typically only care about the project, which is the fixture. In testing handlers, you care about the world, so it's both the fixture and itself.